web/templates/page/about.html.eex

```diff
@@ -1,115 +1,82 @@
 <div class="ma3 ma4-m ma4-l">
   <h2 class="tc">About</h2>
   <div class="bb bw2 bb--hl-dark-blue pb3 pt4">
-    <p class="b">What is Healthlocker?</p>
-
+    <p class="b tc">The Greenwich integrated Children's service ADHD team</p>
     <p>
-      Healthlocker is a secure platform powered by the
-      <%= link "South London and Maudsley NHS Foundation Trust (the trust)", to: "http://www.slam.nhs.uk/",
-          class: "hl-aqua flex-wrap underline" %>
-      that promotes supported self-management and opportunities to improve
-      communication between service users, carers and clinicians.
-    </p>
-
-    <p class="b">
-      It is currently in development, more
-      features and improvements will be added in the coming months.
-    </p>
-
-    <div class="tc mv3">
-      <%= img_tag("/images/About_Clinician.png", class: "w4") %>
-    </div>
-  </div>
-
-  <div class="mt4 mb5">
-    <p class="b">Who is Healthlocker for?</p>
-
+    <p class="b">What do we do?</p>
+   <p>
+      The team provide assessment and diagnostic services for children and young people, aged 5-18, in the 
+      Greenwich borough who are presenting with symptoms of ADHD.
+   <p>
+      We also provide support to children and young people with ADHD and their families which includes 
+      psycho-education, parent support, school liaison and, where necessary, medication. The team work 
+      closely with CAMHS and Community Paediatrics.
+  </p>
+  <p>
+    
+  <p class="b">Who can refer to our service?</p>
+   <p>
+      Referrals are accepted from all professionals in healthcare, social care and education.
+   </p>
+   <p>
+     
+  <p class="b">Who is this ADHD service for?</p>
+<ul>
+  <li>children and young people aged 5-18</li>
+  <li>children and young people registered with a GP in the Greenwich Borough</li>
+  <li>children and young people presenting with a persistent pattern of inattention and / or hyperactivity / impulsivity 
+    that is more frequent and severe than expected compared to children of the same age. These difficulties occur in more 
+    than one setting and are causing impairment to the child's day to day functioning.</li>
+</ul>
+   <p class="i">If a child appears to have complex mental health needs, in addition to possible ADHD, a GP referral should 
+     be made to <%= link "CAMHS", to: "http://oxleas.nhs.uk/services/service/child-and-adolescent-mental-1/",
+          class: "hl-aqua flex-wrap underline" %> in the first instance.
+   <p>  
+        If a child appears to have social communication difficulties, for example possible ASD, a GP referral should be 
+        made to <%= link "Community Paediatrics", to: "http://oxleas.nhs.uk/services/service/community-paediatrics-medical-/",
+        class: "hl-aqua flex-wrap underline" %> in the first instance.
+   </p>
+     
+   <p> Who are the team?</p>
+ Dr Nicola Reynolds, Highly Specialist Clinical Psychologist / Clinical Lead and Team Manager<br>   
```

Comment:
Is this section meant to be in a paragraph as well? Needs to have opening `<p>` at the beginning and closing `</p>` at the end.